pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -1562,6 +1630,11 @@ func TestUserSignupReactivateAfterDeactivated(t *testing.T) {
 				Status: v1.ConditionTrue,
 				Reason: "Deactivated",
 			},
+			v1alpha1.Condition{
+				Type:   v1alpha1.UserSignupUserDeactivatingNotificationCreated,
+				Status: v1.ConditionFalse,
+				Reason: "UserNotInPreDeactivation",
+			},
```

Comment:
Here you verify that the status condition is set when the UserSignup is reactivated, but you don't set the condition to 
```
                {
			Type:   toolchainv1alpha1.UserSignupUserDeactivatingNotificationCreated,
			Status: corev1.ConditionTrue,
			Reason: toolchainv1alpha1.UserSignupDeactivatingNotificationCRCreatedReason,
		}
```
in the `// given` part - analogically as it's done for `UserSignupUserDeactivatedNotificationCreated` condition:
https://github.com/sbryzak/host-operator/blob/6cb26692b97bd178314c1b0afabe12ae6d219d54/pkg/controller/usersignup/usersignup_controller_test.go#L1596-L1600 
to verify that the condition will be reset